ArcGISRuntimeSDKQt_QMLSamples/Layers/WmsLayerUrl/WmsLayerUrl.qml

```diff
@@ -39,8 +39,8 @@ Rectangle {
 
             // Add a WMS Layer by specifying the URL and layer name
             WmsLayer {
-                url: "https://nowcoast.noaa.gov/arcgis/services/nowcoast/radar_meteo_imagery_nexrad_time/MapServer/WMSServer?request=GetCapabilities&service=WMS"
-                layerNames: ["1"]
```

Comment:
wonder why the layer names differed between c++ and qml

Reply:
I'm not sure I understand, it looks like they were both "1" before and now they're both "conus_base_reflectivity_mosaic"?